ironfish/src/network/utils/serializers.ts

```diff
@@ -11,6 +11,8 @@ import { BigIntUtils } from '../../utils/bigint'
 
 export const MINERS_FEE_TRANSACTION_SIZE_BYTES = 562
 
+export const GRAFFITI_SIZE = 32
```

Comment:
This is probably used in a few places outside of the network code, so I'm curious if there's a better place to put this -- maybe exporting it from the block or block header file makes sense? Or creating a new block/blockchain constants file.

Reply:
Would it make sense to export it from `src/utils/graffiti.ts` or should I avoid doing that since it's a utils file?

Reply:
Moved to block file along with the above two constants